### PR TITLE
opusrtp: add extract (from file) option

### PR DESCRIPTION
**Pull request by giavac**

### CODE REVIEW

File: src/opusrtp.c [hunk 1]
```diff
@@ -787,6 +787,72 @@ void write_packet(u_char *args, const struct pcap_pkthdr *header,
...
+  if ((pcap = pcap_open_offline(input_file, errbuf)) == NULL)
+  {
+    fprintf(stderr,"\nError opening dump file\n");
+    return -1;
+  }
+
+  params = malloc(sizeof(state));
+  if (!params) {
+    fprintf(stderr, "Couldn't allocate param struct.\n");
+    return -1;
```
Comment (rillian) [inline src/opusrtp.c]: I've forgotten how the pcap api works. Should we call `pcap_close(pcap);` here and in all the subsequent cleanup clauses?

File: src/opusrtp.c [hunk 2]
```diff
@@ -787,6 +787,72 @@ void write_packet(u_char *args, const struct pcap_pkthdr *header,
   }
 }
 
+int extract(const char* input_file)
```
Comment (rillian) [inline src/opusrtp.c]: This function, along with references to it and the `--extract` option need to be protected by `#ifdef HAVE_PCAP`. That's what the travis failure is about, and we don't want to require libpcap for all of opus-tools to build, so this needs to remain optional code.

**Comment by giavac** [comment]
Thanks for the review. I will address the requests.

**Comment by rillian** [comment]
Merged upstream in 350617f55d9154f2e95d65687a0a719483d745a7.

**Comment by giavac** [comment]
Thank you.

**Pull request closed without merging**